client/nearby/src/components/NeabyService.jsx

```diff
@@ -4,11 +4,7 @@ import WorkspaceContainer from './WorkspaceContainer';
 
 const NearbyService = () => {
     return window.__initialData__ ? (
-        <WorkspaceContainer
-            nearbyWorkspaces={window.__initialData__.nearbyWorkspaces}
-            allWorkspaceInfo={window.__initialData__.allWorkspaceInfo}
-            photos={window.__initialData__.photos}
-        />
+        <WorkspaceContainer nearbyWorkspaces={window.__initialData__.nearbyWorkspaces} />
```

Comment:
Interesting pattern here. I like the idea of initial data, I don't think it should belong on a property with `__ __` because that is usually reserved for the prototype

Reply:
I agree. I think I remember seeing the underscores in an article on ssr with react and just followed along exactly to make it easier but since things are working now it would look better if I changed it to just `window.initialData`